support-workers/src/main/scala/com/gu/stripe/Stripe.scala

```diff
@@ -39,40 +37,16 @@ object Stripe {
   }
 
   object StripeList {
-    implicit def codec[T](implicit encoder: Encoder[T], decoder: Decoder[T]): Codec[StripeList[T]] = deriveCodec[StripeList[T]]
-  }
-
-  case class StripeList[T](total_count: Int, data: Seq[T]) extends StripeObject
-
-  object Source {
```

Comment:
source and customer are now different depending whether it's from payment method or token.